EMacco/atm-simulation
# #168778853 Withdraw Page

#### What does this PR do?
This PR allows a user to be able to withdraw from his account

#### Description of Task to be completed?
- Create navigation to withdrawal page
- Validate amount
- Show receipt of withdraw

#### How should this be manually tested?
- Checkout to this branch
- Run foreman start
- Open your browser and go to http://localhost:3000
- Login
- Click on Withdraw on the profile card
- Enter an amount
- You should see that your account has been debited

#### What are the relevant pivotal tracker stories?
[#168778853](https://www.pivotaltracker.com/story/show/168778853)
